src/index.html

```diff
@@ -9,10 +9,63 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/styles.scss"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card_image"
+        src="images/imac.jpeg"
+        alt="Imagem IMAC"
+      />
+      <div class="card_content">
+        <h1 class="card_title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card_product_code">Product code: 195434</p>
+
+        <div class="card_reviews">
+          <div class="card_stars">
```

Comment:
Tani, the ideia is to use the code from the task Stars here, just copying as it is, so it can be easily changed, as it worked there.
In this way, you should rename this block to `card__stars stars stars--4`, that way you are saying that _Stars_ is a -Block_ that is also an _Element_ of the _Block_ _Card_, and with the _Modifier_ "4" to represent 4 stars.
And them, all card_stars__star could be just renamed to `stars__star`

Reply:
Ok, thank you for the explanation!!! 😄